dev-proxy/InactivityTimer.cs

```diff
@@ -0,0 +1,30 @@
+using DevProxy.Abstractions;
+
+namespace DevProxy
+{
+    public class InactivityTimer : IInactivityTimer
+    {
+        private Timer? _timer;
```

Comment:
Since timer is created in the constructor, let's not mark it as nullable. This will help us simplify our code.